src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Integer.Signed.cs

```diff
@@ -127,27 +127,27 @@ public static bool TryParse(ReadOnlySpan<byte> source, out short value, out int
         /// </exceptions>
         public static bool TryParse(ReadOnlySpan<byte> source, out int value, out int bytesConsumed, char standardFormat = default)
         {
-            switch (standardFormat)
-            {
-                case default(char):
-                case 'g':
-                case 'G':
-                case 'd':
-                case 'D':
-                    return TryParseInt32D(source, out value, out bytesConsumed);
+            int standardFormatLowercase;
 
-                case 'n':
-                case 'N':
-                    return TryParseInt32N(source, out value, out bytesConsumed);
+            if (standardFormat == default
```

Comment:
This helps, but it would be even better to have overload of the API that does not have the standardFormat argument at all.